docs/source/error_code_list2.rst

```diff
@@ -547,7 +547,7 @@ Correct usage:
 .. code-block:: python
 
     # Use "mypy --enable-error-code unimported-reveal"
-    from typing import reveal_type   # or `typing_extensions`
+    from typing import reveal_type   # "from typing_extensions" in Python 3.10 and earlier
```

Comment:
I think this one can be left as-is: the existing message is more concise, and isn't incorrect (you can still import it from `typing_extensions` on 3.11+)